pkgs/development/tools/qtcreator/default.nix

```diff
@@ -8,30 +8,31 @@ with lib;
 let
   # Fetch clang from qt vendor, this contains submodules like this:
   # clang<-clang-tools-extra<-clazy.
-  clang_qt_vendor = llvmPackages_8.clang-unwrapped.overrideAttrs (oldAttrs: {
+  clang_qt_vendor = llvmPackages_10.clang-unwrapped.overrideAttrs (oldAttrs: {
     src = fetchgit {
-      url = "https://code.qt.io/clang/clang.git";
-      rev = "c12b012bb7465299490cf93c2ae90499a5c417d5";
-      sha256 = "0mgmnazgr19hnd03xcrv7d932j6dpz88nhhx008b0lv4bah9mqm0";
+      url = "https://code.qt.io/clang/llvm-project.git";
```

Comment:
the repository for their vendored llvm/clang changed, the old one is marked deprecated